src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { nowUOrange } from '../../assets';
+import {
+    topNavLinks,
+    bottomNavLinks,
+    mobileNavLinks,
+} from '../../utils/constants';
+import NavLinkSection from './navLinkSection/NavLinkSection';
+import NavDropdown from './navDropdown/NavDropdown';
+import classes from './Navbar.module.scss';
+
+// import NewsletterSignupModal from '../newsletter-signup-modal/NewsletterSignupModal';
+
+const Navbar = ({ newsModal }) => {
+    const [dropdownNavDisplay, setDropdownNavDisplay] = useState(false);
+    const openDropdownNav = () => setDropdownNavDisplay(true);
+    const closeDropdownNav = () => setDropdownNavDisplay(false);
+
+    const [showNewsModal, setShowNewsModal] = useState(false);
+    const toggleNewsModal = () => {
```

Comment:
This doesn't toggle the newsletter signup modalas it can't be clicked if the modal is showing. A better name would be `popNewsletterSignupModal` or `openNewsletterSignupModal`.